<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="A2ML30049" name="io.mosip.e2e.tests.AuthenticationTests" tests="1" failures="0" timestamp="15 Jul 2019 12:27:26 GMT" time="1831.518" errors="1">
  <testcase name="authenticateTest" time="1831.518" classname="io.mosip.e2e.tests.AuthenticationTests">
    <error type="io.mosip.e2e.util.TestRigException" message="IDA Authentication Failed Because all uins didn&#039;t get generated">
      <![CDATA[io.mosip.e2e.util.TestRigException: IDA Authentication Failed Because all uins didn't get generated
at io.mosip.e2e.tests.AuthenticationTests.authenticateTest(AuthenticationTests.java:17)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at io.mosip.e2e.runner.EndToEndRun.main(EndToEndRun.java:28)
]]>
    </error>
  </testcase> <!-- authenticateTest -->
</testsuite> <!-- io.mosip.e2e.tests.AuthenticationTests -->
